gridstatus/ercot_api/ercot_api.py

```diff
@@ -1068,7 +1068,7 @@ def get_historical_data(
                     break  # Exit the loop if the operation is successful
 
                 except Exception as e:
-                    if "Rate limit is exceeded" in response.decode("utf-8"):
```

Comment:
I ran into an issue when running tests that the `response` couldn't be decoded. The exception message can be used to determine if we are being rate-limited.